vendors/espressif/boards/esp32s2/aws_demos/config_files/aws_demo_config.h

```diff
@@ -46,7 +46,7 @@
  *
  *  These defines are used in iot_demo_runner.h for demo selection */
 
-#define CONFIG_CORE_MQTT_MUTUAL_AUTH_DEMO_ENABLED
+#define CONFIG_CORE_HTTP_S3_DOWNLOAD_MULTITHREADED_DEMO_ENABLED
```

Comment:
Are we enabling this by default for esp32s2 or was this checked in by mistake?

Reply:
Check in by mistake, I will rever it